documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -4,12 +4,12 @@
 Installation by Source Using Anaconda Environment for Unix-based Systems: Linux and Mac OSX
 *******************************************************************************************
 
-#. Download and install the `Anaconda Python Platform <https://www.anaconda.com/download/>`_ for Python 3.7.
+#. Download and install `Anaconda Individual Edition <https://www.anaconda.com/products/individual/>`_ for Python 3.8.
```

Comment:
Can you also remove "Python 3.8" from this line? Only python 3 versions are available at the link, and the environment sets the python version anyways (in our case it is actually python 3.7). This will avoid possible confusion and prevent us from having to update this in the future

Reply:
@amarkpayne everything is done